File: median of array/median.py
# median of arrays
def median(num1,num2):
    num=num1+num2
    num.sort()
    if len(num)%2==0:
        return (num[len(num)//2]+num[len(num)//2-1])/2
    else:
        return num[len(num)//2]
num1=[1,3]
num2=[2]
print(median(num1,num2))

File: median of array/median.js
//Median of Two Sorted Arrays
function med(num1,num2){
    let nums=num1.concat(num2);
    nums.sort((a,b)=>a-b);
    
    let len=nums.length;
    
    if(len%2===0){
        return (nums[len/2]+nums[(len/2)-1])/2;
    }
    else{
        return nums[Math.floor(len/2)];
    }
}
console.log(med([1,2],[3,4]));
console.log(med([1,3],[2]));